docs/reboot-node-using-kured.md

```diff
@@ -0,0 +1,9 @@
+# Testing Kured
+
+To do a manual test connect to a node you want to restart within the cluster and create a file on the node itself.
```

Comment:
Can you add something about how to connect to the node i.e. `kubectl exec` into the kured pod ?